lib/ui/home.dart

```diff
@@ -160,7 +160,7 @@ class _HomeState extends State<Home> with WidgetsBindingObserver {
 
   Future<void> _retrieveDynamicLink(HomeViewmodel viewModel) async {
     final PendingDynamicLinkData data =
-        await FirebaseDynamicLinks.instance.retrieveDynamicLink();
+        await FirebaseDynamicLinks.instance.getInitialLink();
```

Comment:
looks good 👍

Reply:
@DavidFD005 you will need this change for app entry..

Reply:
I've already changed this :) because I've update the firebase dynamic link lib